pkg/bootkube/create.go

```diff
@@ -80,15 +80,19 @@ func createAssets(manifestDir string) error {
 		return err
 	}
 
-	includeExtendedAPIs := false
-	mapper, typer := f.Object(includeExtendedAPIs)
+	// aaron: is this ok that we can't specify the extend api option anymore?
```

Comment:
This is almost a verbatim copy of the kubectl implementation of `create -f manifestDir`. I'd re-check those changes if you haven't. Seems like as long as we support the same as that command we should be okay.